src/document.rs

```diff
@@ -85,14 +83,14 @@ impl PopplerDocument {
             }
         }
     }
-    pub fn get_permissions(&self) -> u8 {
-        unsafe { sys_doc::poppler_document_get_permissions(self.0) as u8 }
+    pub fn get_permissions(&self) -> sys_doc::PopplerPermissions {
+        unsafe { sys_doc::poppler_document_get_permissions(self.0) }
     }
 
-    pub fn get_n_pages(&self) -> usize {
+    pub fn get_n_pages(&self) -> c_int {
         // FIXME: what's the correct type here? can we assume a document
         //        has a positive number of pages?
-        (unsafe { sys_doc::poppler_document_get_n_pages(self.0) }) as usize
+        unsafe { sys_doc::poppler_document_get_n_pages(self.0) }
```

Comment:
I also don't know, but I figured it would be more conservative to forward the original type
But I think it also should be ok to try a conversion into an usize, and then possibly failing on negative numbers. But meh..